release/gcb/cloud_builder.sh

```diff
@@ -75,6 +75,33 @@ add_license_to_tar_images "${REL_DOCKER_HUB}" "${CB_VERSION}" "${OUTPUT_PATH}"
 # log where git thinks the build might be dirty
 git status
 
+#Handle CNI artifacts.
+pushd ../cni
+CNI_OUT=$(make DEBUG=0 where-is-out)
+# CNI version strategy is to have CNI run lock step with Istio i.e. CB_VERSION
+MAKE_TARGETS=(build)
+VERBOSE=1 DEBUG=0 ISTIO_DOCKER_HUB="${CB_ISTIOCTL_DOCKER_HUB}" HUB="${CB_ISTIOCTL_DOCKER_HUB}" VERSION="${ISTIO_VERSION}" TAG="${ISTIO_VERSION}" make "${MAKE_TARGETS[@]}"
+BUILD_DOCKER_TARGETS=(docker.save)
+
+if [ -d "${CNI_OUT}/docker" ]; then
+    rm -r "${CNI_OUT}/docker"
+fi
+
+VERBOSE=1 DEBUG=0 ISTIO_DOCKER_HUB=${REL_DOCKER_HUB} HUB=${REL_DOCKER_HUB} VERSION="${ISTIO_VERSION}" TAG="${ISTIO_VERSION}" make "${BUILD_DOCKER_TARGETS[@]}"
+
+mkdir "/cni_tmp"
```

Comment:
should this be in /tmp instead?

Reply:
No - I want this to a new directory the "add_license_to_tar_images" adds the licenses to a all tar files in a directory so I wanted a clean directory with no possibility of any other tar files.